src/bat/inswin31.bat

```diff
@@ -1,8 +1,12 @@
 @echo off
 unix -s dosemu-preinstallwin31
 if errorlevel 1 goto faildownload
+%USERDRV%:\tool\stuffkey\stuffkey.com /F:f:\stuffkey\w31skbsw.txt
 if exist %USERDRV%:\inst\win31\setup.exe %USERDRV%:\inst\win31\setup.exe /h:f:\win31\dosemu.shh
 if errorlevel 1 goto failwinsetup
+if exist %USERDRV%:\inst\win31\install.exe %USERDRV%:\inst\win31\install.exe /h:f:\win31\dosemu.shh

```

Comment:
I don't think you should check
the existence of everything.
Its an installer script, so it should
just call the installer, and if missing -
user must see `bad command or file name`.
Otherwise you "eat" the errors.

Reply:
Yes, makes sense, I'll change that. But I'll have to think a bit about the best approach. Almost all versions use `setup.exe`. So far I only found the French one to lack `setup.exe` and having `install.exe` instead.

Maybe I could invert the checks such that if there is no `install.exe` there _must_ be a `setup.exe` and vice-versa. Then you will get a command not found when neither are there. I just hope there's no third option that I haven't come across yet.

Reply:
I updated this now.